packages/site/src/routes/_docs.svtext

```diff
@@ -662,21 +662,34 @@ Frontmatter also interacts with layouts, you can find more details in the [Layou
 
 ## Integrations
 
-### With Sapper
+### With shiki
 
-To use mdsvex with sapper you need to add the mdsvex configuration to both the client and server sections of the rollup or webpack configuration. You will also need to add the CLI argument `--ext '.svelte .svx'` to all of the sapper scripts (`dev`, `build`, and `export`) in order to tell sapper that it should also allow `.svx` files to be page routes.
+You can use shiki for highlighting rather than prism by leveraging the `highlighter` option:
 
-Or you can use the templates:
-- [Rollup](https://github.com/pngwn/sapper-mdsvex-template)
+  ```js
+import { mdsvex, escapeSvelte } from 'mdsvex';
+import { createHighlighter } from 'shiki';
 
-  ```bash
-  npx degit "pngwn/sapper-mdsvex-template" my-app
-  ```
-- [Webpack](https://github.com/shiryel/sapper-mdsvex-template-webpack)
+const theme = 'github-dark';
+const langs = ['javascript', 'typescript'];
+const highlighter = await createHighlighter({
+	themes: [theme],
+	langs
+});
+for (const lang of langs) {
+	await highlighter.loadLanguage(lang);
+}
```

Comment:
I don't think it's not need to load the languages again because it was created with that

Reply:
@benmccann do we want to load the languages on demand in this example or do you think it is better to keep it simple?

Reply:
great catch @honnip! that was in the example I copied from, but I took a look at the docs and you look to be correct that it's unnecessary, so I've removed it